server/common/comments/get.ts

```diff
@@ -20,21 +24,31 @@ export default async function get(req: NextApiRequest, res: NextApiResponse) {
     if (proposalId) filters.proposalId = +proposalId;
     if (deliverableId) filters.deliverableId = +deliverableId;
     if (userId) filters.userId = +userId;
+    if (type) filters.type = type;
```

Comment:
getting type should have the same reasoning that "when im fetching for deliverable comments i should also get the reviews for that deliverable"

Reply:
I can add to make it clearer, but this logic is implicit when I search only for the deliverableId. All comments of type 'deliverable' and 'review' come in this get.